tests/integration_tests/kafka_simple_include_start_ts/run.sh

```
@@ -0,0 +1,128 @@
+        if str(row_ts) != str(expected_ts):
+            raise SystemExit("row expected_start_ts %s != %s" % (row_ts, expected_ts))
+    elif start_ts is not None:
+        raise SystemExit("startTs must be omitted, got %r" % (start_ts,))
+if found < 2:
+    raise SystemExit("found %d probe INSERT messages, want 2" % found)
+print("probe ok: %d INSERT messages, startTs present=%s ts=%s" % (found, want, expected))
+PY
+}
+
+insert_probe_rows() {
+	mysql -uroot -h${UP_TIDB_HOST} -P${UP_TIDB_PORT} --default-character-set utf8mb4 -N <<EOF
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)